content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -11,14 +11,12 @@ weight: 30
 
 {{% capture overview %}}
 
-This document describes the concept of `StorageClass` in Kubernetes. Familiarity
+This document describes the concept of a StorageClass in Kubernetes. Familiarity
 with [volumes](/docs/concepts/storage/volumes/) and
 [persistent volumes](/docs/concepts/storage/persistent-volumes) is suggested.
 
 {{% /capture %}}
 
-{{< toc >}}
```

Comment:
Can we keep the toc?

Reply:
It's added automatically, without explicitly specifying it.

Reply:
You can see it here if you like: https://deploy-preview-10041--kubernetes-io-master-staging.netlify.com/docs/concepts/storage/storage-classes/

Reply:
Ah ok, I was confusing it with the Volumes page, which has a TOC-like list of all the volume types.